examples/case_studies/hierarchical_partial_pooling.ipynb

```diff
@@ -4,11 +4,12 @@
    "cell_type": "markdown",
```

Comment:
<p>The <code>notebook_name</code> piece should not be copied literally. It needs to be specific to the notebook. You can use the notebook filename without the extension for example. https://docs.pymc.io/en/latest/contributing/jupyter_style.html#first-cell</p><p> </p><p>I also see you have added Chris as author,  but there is no authorship section towards the bottom of the notebook. https://docs.pymc.io/en/latest/contributing/jupyter_style.html#authorship-and-attribution </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/393/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thanks for the review! I have changed the name, please let me know if I got that right. I deleted the authorship because I couldn't find the original commit that I thought I saw from Chris as this file's origin. I can add that back in if that was a correct attribution.</p>